openqa-label-known-issues

```diff
@@ -28,15 +28,16 @@ MOJO_CONNECT_TIMEOUT=${MOJO_CONNECT_TIMEOUT:-30}
 client_args=(api --header 'User-Agent: openqa-label-known-issues (https://github.com/os-autoinst/scripts)' --host "$host_url" --retries="$retries")
 
 out="${REPORT_FILE:-$(mktemp -t openqa-label-known-issues--output-XXXX)}"
+declare timeago
 trap 'error-handler "$LINENO"' ERR
 trap 'test "$KEEP_REPORT_FILE" == "1" || rm "$out"' EXIT
+trap 'test "$KEEP_JOB_HTML_FILE" == "1" || rm "$html_out"' EXIT
```

Comment:
If you have multiple handlers for the same signal, the last one will be winning and just overwrite the previous ones.